notebooks/02_model/cornac_bpr_deep_dive.ipynb

```diff
@@ -0,0 +1,595 @@
+{
```

Comment:
<p>couple grammatical suggestions:</p><p> </p><p>The task of personalized ranking aims at providing each user a ranked list of items (recommendations). It is very common the scenarios where recommender systems dealing with user implicit behavior (e.g. purchases, clicks).</p><p> </p><p>-&gt;</p><p> </p><p>The task of personalized ranking aims at providing each user a ranked list of items (recommendations). This is very common in scenarios where recommender systems are based on implicit user behavior (e.g. purchases, clicks).</p><p> </p><p>-----</p><p>From the Bayesian perspective, BPR maximizes the posterior probability over the model paramters</p><p> </p><p>typo in last word 'paramters'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/937/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Thank you for your suggestions. 
I've updated the writeup.